### PR TITLE
SPIKE: upgrade webdriver dep

### PR DESCRIPTION
**Pull request by akfernun**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
```
Comment (akfernun) [inline .vscode/launch.json:5]: remove this and add `.vscode` to the gitignore

Comment (kellyselden) [reply .vscode/launch.json:5]: you can add it your your system/user level .gitignore. then you don't have to add it to this or any repo.

File: packages/page-objects/src/element.js [hunk 2]
```diff
@@ -74,6 +74,12 @@ class Element extends BaseElement {
   async selectByVisibleText() {
     await this._browser.selectByVisibleText(this._selector, ...arguments);
   }
+
+  async shadow$(selector) {
```
Comment (akfernun) [inline packages/page-objects/src/element.js:78]: This is WIP code and not meant to be "working"

File: .vscode/launch.json (hunk 1 above)
Comment (kellyselden) [reply .vscode/launch.json]: Not necessarily. This repo doesn't care what IDE you use.

**Comment by kellyselden** [comment]
Accomplished in https://github.com/CrowdStrike/faltest/pull/688. Thanks for the inspiration!

**Pull request closed without merging**